clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -72,7 +72,13 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
     return [];
   };
 
-  const { currentLocale } = useI18n();
+  const { currentLocale, setCurrentLocale } = useI18n();
+
+  useEffect(() => {
+    if (!currentLocale && i18n.locale) {
+      setCurrentLocale(i18n.locale);
+    }
+  }, [currentLocale, i18n.locale, setCurrentLocale]);
```

Comment:
FWIW: seems like the move is to take all the `i18n` props and hoist those into context. I'm not opposed to that at all, it was just very _useful_ to me at the time to use prop-drilling to ensure I found everything 👍

Reply:
Kind of depends how close we want to continue to keep things to the Linugi library. They inject the i18n object into the Provider to use, but then they update it using an event emitter. We would need to write a similar emitter to subscribe to https://github.com/lingui/js-lingui/blob/main/packages/react/src/I18nProvider.tsx#L74

If we are abandoning the idea of being Lingui compatible, I'd just as soon convert the entire i18n-utils to the context.